babbage/model/dimension.py

```diff
@@ -49,6 +49,10 @@ def cardinality_class(self):
                 return 'low'
             return 'tiny'
 
+    @property
+    def datatype(self):
```

Comment:
I documented this now that I implemented it in the superclass Concept. Does it make sense to also document here?